app/safe/internal/state/state.go

```diff
@@ -12,16 +12,17 @@ import (
 	"bytes"
 	"encoding/base64"
 	entity "github.com/shieldworks/aegis/core/entity/data/v1"
+	"github.com/shieldworks/aegis/core/env"
 	"github.com/shieldworks/aegis/core/log"
-	"sync"
 	"time"
 )
 
 const InitialSecretValue = `{"empty":true}`
 const BlankAgeKeyValue = "{}"
 
+// ageKey is set only once during initialization; we don’t need to lock
+// access to it.
 var ageKey = ""
-var lock sync.Mutex
```

Comment:
Got rid of this semaphore; race condition is near impossibile. and when it happens, something going awry is very very not possible as well.